test/vmtests/vmtests/test_no_change.py

```diff
@@ -83,5 +103,17 @@ def test_no_change(
         with open(os_release_path, "r") as os_release_fd:
             os_release_text = os_release_fd.read()
 
-            assert "ID=mariner" in os_release_text
-            assert 'VERSION_ID="2.0"' in os_release_text
+            assert ("ID=azurelinux" in os_release_text) or ("ID=mariner" in os_release_text)
```

Comment:
The test should know which image it is testing against and therefore should know exactly which string is correct.